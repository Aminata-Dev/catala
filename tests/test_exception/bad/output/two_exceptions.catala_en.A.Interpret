[ERROR] There is a conflict between multiple validd consequences for assigning the same variable.

This consequence has a valid justification:
  --> test_exception/bad/two_exceptions.catala_en
   | 
15 |   definition x equals 2
   |                       ^
   + Test

This consequence has a valid justification:
  --> test_exception/bad/two_exceptions.catala_en
   | 
12 |   definition x equals 1
   |                       ^
   + Test
