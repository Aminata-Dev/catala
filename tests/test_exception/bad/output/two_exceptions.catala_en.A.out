[ERROR] There is a conflict between multiple exceptions for assigning the same variable.

This justification is true:
  --> test_exception/bad/two_exceptions.catala_en
   | 
15 |   definition x equals 2
   |              ^
   + Test

This justification is true:
  --> test_exception/bad/two_exceptions.catala_en
   | 
12 |   definition x equals 1
   |              ^
   + Test
